alf/algorithms/muzero_algorithm.py

```diff
@@ -407,11 +495,13 @@ def preprocess_experience(self, root_inputs: TimeStep,
                     batch_info=batch_info,
                     replay_buffer=replay_buffer)
                 exp = self._data_transformer.transform_experience(exp)
-                observation = exp.observation
+                observation = _unfold1(exp.observation)
```

Comment:
This should also use `_unfold1_adapting_episode_ends`. The previous code does not do capping, which is not right.

Reply:
You are right. Fixed.